test/e2e/cypress/e2e/saptune_details.cy.js

```diff
@@ -0,0 +1,113 @@
+describe('Saptune Details page', () => {
+  beforeEach(() => {
```

Comment:
Do we need to visit the page before every test?
Or just visiting once would work?
This way, we would be testing the reactivity better I guess

Reply:
Hey man so you mean instead of using beforeEach, using before?
After a quick research using beforeEach will isololate and increase the independence of the test, but on the other side it may increase the run time of the test itself. 

For me both are working fine, should we stick with before as we use it in many other tests?  :D

Reply:
I would use `before`, as our page is a reactive app, and we want to see that changes in life.
Being independent or not in this case is not something I see valuable, as we don't change page during the tests